src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1460,6 +1462,13 @@ private void NotifyFeaturesAboutStateChange(DatabaseRecord record, long index)
 #endif
                                 }
                             }
+                            catch (OperationCanceledException)
```

Comment:
this can only happen if context pool is being disposed, so when server is shutting down. Shouldn't we just ignore it?

Reply:
That's what I'm doing here.

Reply:
Yes, sorry. It's late :)